data/templates/template.typst

```diff
@@ -16,36 +16,62 @@
   keywords: (),
   date: none,
   abstract: none,
+  thanks: none,
   cols: 1,
   margin: (x: 1.25in, y: 1.25in),
   paper: "us-letter",
   lang: "en",
   region: "US",
   font: (),
   fontsize: 11pt,
+  mathfont: "New Computer Modern Math",
+  codefont: "DejaVu Sans Mono",
+  linestretch: 1,
   sectionnumbering: none,
+  linkcolor: ["#800000"],
+  citecolor: ["#0000FF"],
+  toccolor: ["#800000"],
   doc,
 ) = {
   set document(
     title: title,
-    author: authors.map(author => content-to-string(author.name)),
+    author: authors.map(author => content-to-string(author.name)).join(", ", last: ", and "),
     keywords: keywords,
   )
   set page(
     paper: paper,
     margin: margin,
     numbering: "1",
   )
-  set par(justify: true)
+  set par(
+    justify: true,
+    leading: linestretch * 0.65em
+  )
   set text(lang: lang,
            region: region,
            font: font,
            size: fontsize)
+  show math.equation: set text(font: mathfont)
+  show raw: set text(font: codefont)
   set heading(numbering: sectionnumbering)
 
+  show link: set text(fill: rgb(content-to-string(linkcolor)))
+
+  show link: this => {
+    if type(this.dest) == label {
+        text(this, fill: rgb(content-to-string(filecolor)))
+    }
+  }
+  show ref: this => {
+    text(this, fill: rgb(content-to-string(citecolor)))
+  }
+
   if title != none {
     align(center)[#block(inset: 2em)[
-      #text(weight: "bold", size: 1.5em)[#title]
+      #text(weight: "bold", size: 1.5em)[#title #if thanks != none {
+            footnote(thanks, numbering: "*")
+            counter(footnote).update(n => n - 1)
```

Comment:
I assume this is needed because otherwise the first note would be `2`?  That's a bit unexpected when `numbering` is explicitly specified, but maybe Typst works that way?  (I'd consider it a bug if so.)